tests/object/tree/read.c

```diff
@@ -73,3 +73,59 @@ void test_object_tree_read__two(void)
 	git_object_free(obj);
 	git_tree_free(tree);
 }
+
+#define BIGFILE "bigfile"
+
+void test_object_tree_read__largefile(void)
+{
+	git_reference *ref;
+	git_commit *commit;
+	git_tree *tree;
+	git_oid oid;
+	const git_tree_entry *entry;
+	git_object *object;
+	git_buf file = GIT_BUF_INIT;
+	int fd;
+	fstore_t prealloc;
+	git_index *idx;
+
+	if (!cl_is_env_set("GITTEST_INVASIVE_FS_SIZE"))
```

Comment:
This reminds me that we still ain't got this environment variable set on Azure

Reply:
Filed #4933 for that.